### PR TITLE
Add page container component for all page.

### PR DESCRIPTION
**Pull request by yanguoyu**
Only need to attention to the page head. Others in the overview will open another pr.

https://user-images.githubusercontent.com/12881040/224863244-67af3724-bb37-4e3f-a5a9-b0ff6ca4ed32.mov

### CODE REVIEW

**Comment by Keith-CY** [comment]
> Only need to attention to the page head. Others in the overview will open another pr.

The arrow should point down by default and point up on hovering
image
image

**Comment by yanguoyu** [comment]
> > Only need to attention to the page head. Others in the overview will open another pr.
> 
> The arrow should point down by default and point up on hovering image image

Fixed.

image
image

File: packages/neuron-wallet/src/controllers/api.ts [hunk 1]
```diff
@@ -260,10 +260,20 @@ export default class ApiController {
...
+        result: nativeTheme.shouldUseDarkColors
+      }
     })
 
+    handle('set-theme', async (_, theme: 'system' | 'light' | 'dark') => {
+      SettingsService.getInstance().themeSource = theme
+      return {
+        status: ResponseCode.Success
+      }
+    })
```
Comment (devchenyan) [inline packages/neuron-wallet/src/controllers/api.ts:275]: `enum ThemeEnum {
    SYSTEM = 'system',
    LIGHT = 'light',
    DARK = 'dark'
}`

File: packages/neuron-ui/src/widgets/Tooltip/index.tsx [hunk 2]
```diff
@@ -1,24 +1,49 @@
...
     return (
       <div
         className={`${styles.tipWithString} ${className || ''}`}
         data-tooltip={tip}
         data-placement={placement}
         data-placement-center={center}
+        data-trigger={trigger}
+        onClick={trigger === 'click' ? onChangeIsTipShow : undefined}
+        data-tip-show={isTipShow}
+        onKeyPress={undefined}
```
Comment (devchenyan) [inline packages/neuron-ui/src/widgets/Tooltip/index.tsx:41]: use `className = '' ` to avoid `className || ''` everywhere

**Comment by yanguoyu** [comment]
@JeffreyMa597 @devchenyan Please have a review.

**Review by devchenyan — APPROVED** [review]

File: packages/neuron-wallet/src/controllers/api.ts (hunk 1 above)
Comment (yanguoyu) [reply packages/neuron-wallet/src/controllers/api.ts:275]: Are there any differences between these? Or why this is better?

**Review by Keith-CY — APPROVED** [review]

**Pull request merged**